inc/saitunnel.h

```diff
@@ -1149,8 +1171,6 @@ typedef struct _sai_tunnel_api_t
     sai_remove_tunnel_map_entry_fn               remove_tunnel_map_entry;
     sai_set_tunnel_map_entry_attribute_fn        set_tunnel_map_entry_attribute;
     sai_get_tunnel_map_entry_attribute_fn        get_tunnel_map_entry_attribute;
-    sai_bulk_object_get_attribute_fn             get_tunnels_attribute;
```

Comment:
why this was removed ?

Reply:
Corrected it.
I removed it cause i did not get the method definition in saitunnel.h, and thought those method should not be needed.

I got those two types def in the saitype.h.

thanks for pointing it out.